### ISSUE TITLE
ExtendViewIntoTitleBar and keeping controls under TitleBar interactible

### ISSUE THREAD

**Issue by milos12345** [Kinnara/ModernWpf#16]
If we use `ui:TitleBar.ExtendViewIntoTitleBar ="True"` but have a control, like a button at the top (overlapping with TitleBar), the element cannot be interacted with as TitleBar handles it. Is it possible to have just a right section of TitleBar perform its original task with 3 buttons and section for moving the window?

**Comment by Kinnara** [comment]
Setting `WindowChrome.IsHitTestVisibleInChrome="True" ` on the control should do the trick.

**Comment by milos12345** [comment]
Hmm, adding this prevents window resizing. The grips don't show up anymore.

**Comment by Kinnara** [comment]
Good catch. I'll try to find a fix. Notice that you don't have to set the property on the whole title bar area, but just on the interactive controls. This way at least there's still somewhere to grip.

**Comment by niels9001** [comment]
@Kinnara Thanks for the great tip to set : WindowChrome.IsHitTestVisibleInChrome="True"

Please add that to the Readme.md - I was fiddling around for 2 hours to get buttons to be interactive in the titlebar. Should have looked at the issues first!
